.gitignore

```diff
@@ -1 +1,2 @@
-_site
\ No newline at end of file
+_site
+.DS_Store
```

Comment:
Shouldn't this go into a [global gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) as opposed to a project gitignore file?

Reply:
Yup, sorry about that! Should be fixed now.